core/src/main/java/hudson/model/Slave.java

```diff
@@ -200,7 +200,6 @@ public Slave(@NonNull String name, String nodeDescription, String remoteFS, int
         getAssignedLabels();    // compute labels now
 
         this.nodeProperties.replaceBy(nodeProperties);
-         Slave node = (Slave) Jenkins.get().getNode(name);
```

Comment:
Any chance it is used for any kind of initialization? Unlikely, but just in case

Reply:
I checked it and there isn't any.

Reply:
Thanks!